modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/test/I18nServletTest.java

```diff
@@ -74,21 +74,23 @@ public static void setUpClass() throws Exception {
 		_availableLocales = _language.getAvailableLocales();
 		_defaultLocale = LocaleUtil.getDefault();
 		_localesEnabled = PropsValues.LOCALES_ENABLED;
+		_hebrewLocale = new Locale("iw", "IL");
```

Comment:
Can you sort this variable? It should go between `_defaultLocale` and `_localesEnabled`.

Reply:
I have added a SF commit to sort the variable, as suggested.